section4/exercises/classMethods.js

```diff
@@ -28,13 +28,36 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
+  changeProtein() {
```

Comment:
There should be an argument passed into the `changeProtein` method according to the instructions.